Nos/Views/Components/Author/AuthorListView.swift

```diff
@@ -36,6 +41,22 @@ struct AuthorListView: View {
         .nosNavigationBar(title: .localizable.mention)
         .onAppear {
             isSearching = true
+
+            guard let relatedAuthors = relatedAuthors else { return }
+            filteredAuthors = relatedAuthors
```

Comment:
I think I'd expect these authors to be sorted by mutual followees, too. Is that part of the requirement?

Reply:
`relatedAuthors` are already sorted before being sent to the `AuthorListView`